test/01LessonOneTest.js

```diff
@@ -5,36 +5,37 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+return true;
 }
 
 function returnFalse() {
     // should return false
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    return big > small;
 }
 
 function lessThan(small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
-
+    return small < big;
 }
 
 function equalTo(a, b) {
     // should return true if the first argument is equal to the second argument
     // and should return true if the first argument is equal to the second argument
-
+    return a===b;
 }
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
 
+    return a!==b;
 }
```

Comment:
@paulrenenichols - just so you don't think I am not reading instructions, I do see the "To run your program, you can run node path/to/PigLatin.js, to run the tests you can run mocha path/to/PigLatin.js." instruction; however, I am not really sure "exactly" how to do this. I do plan on coming tomorrow if I am not able to figure this out tonight though.